applications/plugins/SofaPardisoSolver/extlibs/README.md

```diff
@@ -0,0 +1,13 @@
+README.md
+=========
+
+This plugin requires the following dependencies:
+- gfortran
+- blas
+- lapack
+- gomp
+
+
+In the "extlibs/"" folder, please download the pardiso500-GNU461-X86-64 library.
```

Comment:
The library name (`pardiso500-GNU461-X86-64.so`) won't be the same for everyone, and for every pardiso versions.